### PR TITLE
Updated Contributing Guide

### PR DESCRIPTION
**Pull request by ChaelCodes**
# Overview
Added a short setup guide to the ruby agent contributing guide to give people an idea of how to set it up. If we want to update these instructions or expand them, please let me know. I had to make these up, because there weren't any install instructions that I could find in the README, Contrib guide, or the Ruby Agent documentation site.

Also fixed broken link.

# Related Github Issue
Include a link to the related GitHub issue, if applicable

# Testing
The agent includes a suite of unit and functional tests which should be used to
verify your changes don't break existing functionality. These tests will run with 
Github Actions when a pull request is made. More details on running the tests locally can be found 
[here for our unit tests](https://github.com/newrelic/newrelic-ruby-agent/blob/main/test/README.md), 
and [here for our functional tests](https://github.com/newrelic/newrelic-ruby-agent/blob/main/test/multiverse/README.md).
For most contributions it is strongly recommended to add additional tests which
exercise your changes. 

# Reviewer Checklist
- [ ] Perform code review
- [ ] Add performance label
- [ ] Perform appropriate level of performance testing
- [ ] Confirm all checks passed
- [ ] Add version label prior to acceptance

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -85,6 +84,20 @@ release from this repo, use the appropriate tag. New Relic usually pushes beta
...
+1. Fork and clone the repo locally
+    - Fork the repository inside GitHub
+    - `git clone git@github.com:<gh username>/newrelic-ruby-agent.git`
+1. Pick a Ruby version
+    - Use rbenv or rvm to install a version of Ruby
+1. Install development dependencies
+    - `bundle install`
+1. Check that your env is setup correctly
+    - bundle exec rake
+
```
Comment (angelatan2) [inline CONTRIBUTING.md:98]: Hello @ChaelCodes,  just a couple of things:
1) Please reword Dev on the title to Development
2) Thanks for adding the bulleted list of steps.  Could you remove the last sentence "If you have any issues setting up the repository, let us know in issues."  There is a link at the end for support.  Typically, when developers run into issues, they comment directly in their respective issues.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,8 +6,7 @@ development team.
 
 When contributing, keep in mind that the agent runs in a wide variety of Ruby
 language implementations (e.g. 2.x.x, jruby, etc.) as well as a wide variety of
-application environments (e.g. Rails, Sinatra, roll-your-own, etc.) See [Ruby agent requirements and supported frameworks]
-(https://docs.newrelic.com/docs/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks)
+application environments (e.g. Rails, Sinatra, roll-your-own, etc.) See [Ruby agent requirements and supported frameworks](https://docs.newrelic.com/docs/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks)
```
Comment (angelatan2) [inline CONTRIBUTING.md:9]: Looks good.

**Review by angelatan2 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -85,6 +84,18 @@ release from this repo, use the appropriate tag. New Relic usually pushes beta
 versions of a release to a working branch before tagging them for General
 Availability.
 
+## Development Environment Setup
+
+1. Fork and clone the repo locally
+    - Fork the repository inside GitHub
+    - `git clone git@github.com:<gh username>/newrelic-ruby-agent.git`
+1. Pick a Ruby version
+    - Use rbenv or rvm to install any version of Ruby (2.7.2 recommended)
```
[the remark this answers is not in the document]
Comment (ChaelCodes) [reply CONTRIBUTING.md]: Updated.

**Review by angelatan2 — APPROVED** [review]
Looks good.  Thank you for contributing to the Ruby Agent!

**Pull request merged**